lib/oauth2-client.js

```diff
@@ -106,7 +100,7 @@ export async function getNewAccessToken({
     const access_token = await _requestAccessToken(
       {url: token_endpoint, body, maxRetries});
```

Comment:
Looks like `maxRetries` is not actually being used inside `__requestAccessToken`. If not used, maybe it can be removed.

Reply:
Removed, thanks.